resources/assets/components/PostFilter/index.js

```diff
@@ -21,6 +21,11 @@ class PostFilter extends React.Component {
                   <option>Accepted</option>
                   <option>Pending</option>
                   <option>Rejected</option>
+                  <option>Good Photo</option>
```

Comment:
I think we should do something more like `<option value="good-photo">Good Photo</option>` This way you don't have to do the `createTagSlug()` work, because you are already passing the slug as the value.

Reply:
ohhhhh yes!